deployments/quick-start/setup.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+
+# Check script dependencies
+for command in http jq
+do
+    if ! hash $command 2>/dev/null; then
+        echo $command is not available, please install $command
+        exit 1
+    fi
+done
+
+# Inspect the deployment file deployments/quick-start/vault.yaml. The deployment starts Vault in development mode with \
+# the root token set to vault-root-token. It is also started using http. In production, you should run Vault over https.
+kubectl apply -f vault.yaml
+
+#2. Setup Vault
+# Wait 5 minutes for vault to be deployed.
+maxWaitSecond=300
+vaultPod=$(kubectl get pods -l app=vault | grep ^vault* |awk '{print $1}')
+while [ $maxWaitSecond -gt 0 ] && [ -z "$vaultPod" ]
+do
+    sleep 1
+    echo waited 1 second for kubernetes to deploy Vault
+    maxWaitSecond=$(($maxWaitSecond-1))
+    vaultPod=$(kubectl get pod | grep ^vault* |awk '{print $1}')
+done
+if [ -z "$vaultPod" ]
+then
+    exit 1
+fi
+
+echo "Vault pod name: $vaultPod"
+# Wait 10 seconds for vault to be running.
+maxWaitSecond=10
+vaultStatus=$(kubectl get pod "$vaultPod" -o=jsonpath='{.status.phase}')
+while [ $maxWaitSecond -gt 0 ] && [ "$vaultStatus" != "Running" ]
+do
+    sleep 1
+    echo waited 1 second for Vault up and running
+    maxWaitSecond=$(($maxWaitSecond-1))
+    vaultStatus=$(kubectl get pod "$vaultPod" -o=jsonpath='{.status.phase}')
+    echo Vault pod status "$vaultStatus"
+done
+if [ "$vaultStatus" != "Running" ]
+then
+    exit 1
+fi
+
+# 2.1 Port forward vault
+nohup kubectl port-forward $vaultPod 8200 &
+
+# Set environment variables and authenticate
+export VAULT_ADDR=http://127.0.0.1:8200
+vault auth vault-root-token
+
+# Set up the Root Certificate Authority
+# Create a Root CA that expires in 10 years: 
+vault mount -path=root-ca -max-lease-ttl=87600h pki
+
+#Generate the root certificate: 
+vault write root-ca/root/generate/internal common_name="Root CA" ttl=87600h exclude_cn_from_sans=true
+
+# Set up the URLs: 
+vault write root-ca/config/urls issuing_certificates="http://vault:8200/v1/root-ca/ca" \
+    crl_distribution_points="http://vault:8200/v1/root-ca/crl"
+
+# 2.3 Create the Intermediate Certificate Authority
+# Create the Intermediate CA that expires in 5 years: 
+vault mount -path=intermediate-ca -max-lease-ttl=43800h pki
+
+# Generate a Certificate Signing Request: 
+http POST http://127.0.0.1:8200/v1/intermediate-ca/intermediate/generate/internal X-Vault-Token:vault-root-token \
```

Comment:
can you use `http POST "${VAULT_ADDR} /v1/intermediate-ca/intermediate/generate/internal X-Vault-Token:${token}"....`